app/_data/docs_nav_ee_1.5.x.yml

```diff
@@ -5,6 +5,11 @@
       url: /introduction
     - text: Key Concepts and Terminology
       url: /introduction/key-concepts
+
+- title: Release Notes
+  icon: /assets/images/icons/documentation/icn-documentation.svg
```

Comment:
Icon didn't really work out, original is too big. Maybe use the icon from the guides and references section as a placeholder, and we can add a new one to the list for Erick to design in the future.